src/main/webapp/styles/itinerary-overview.css

```diff
@@ -1,10 +1,34 @@
-#map{
+#map {
     height: 100%;
+    width: 78%;
+    float: right;
 }
 
+#itinerary-panel{
+    height: 100%;
+    width: 22%;
```

Comment:
Would recommend instead of setting a magic number for the width here (22% is very small on mobile), try to use [Materialize Grid layouts](https://materializecss.com/grid.html).

This way, you can have a layout on desktop that has 4/12 on the left and 8/12 on the right using `<div class="col s12 m4">` and then `<div class="col s12 m8">`, but on mobile, the first div will be on top of the other instead.

Reply:
My map doesn't like being put inside of that. It just doesn't show. It works with the selector though

Reply:
Try adding a div for the map inside of it which has `width: 100%` style.

Reply:
<body onload="initMap()">
    
    <div class="row">
        <div class="col s3">
            <div id="itinerary-panel">
                <div id="mode-panel">
                    <select id="mode">
                        <option value="DRIVING">Driving</option>
                        <option value="WALKING">Walking</option>
                        <option value="BICYCLING">Bicycling</option>
                        <option value="TRANSIT">Transit</option>
                    </select>
                    <label for="mode"><b>Mode of Travel:</b></label>
                </div>
                <select multiple id="waypoints">
                    <option value="Steiner St & Hayes St, San Francisco, CA 94117">Steiner St & Hayes St</option>
                    <option value="Fisherman's Wharf, San Francisco, CA">Fisherman's Wharf</option>
                    <option value="Golden Gate Bridge">Golden Gate Bridge</option>
                    <option value="Presidio of San Francisco, San Francisco, CA">Presidio of San Francisco</option>
                    <option value="Baker Beach, San Francisco, CA">Baker Beach</option>
                </select>
                <label for="waypoints">Locations</label>
            </div>
        </div>
        <div class="col s9">
            <div id="map" style="width: 100%"></div>
        </div>
    </div>

Reply:
try height as well? also try not to use inline styles

Reply:
> ```
> <body onload="initMap()">
> 
> <div class="row">
>     <div class="col s3">
>         <div id="itinerary-panel">
>             <div id="mode-panel">
>                 <select id="mode">
>                     <option value="DRIVING">Driving</option>
>                     <option value="WALKING">Walking</option>
>                     <option value="BICYCLING">Bicycling</option>
>                     <option value="TRANSIT">Transit</option>
>                 </select>
>                 <label for="mode"><b>Mode of Travel:</b></label>
>             </div>
>             <select multiple id="waypoints">
>                 <option value="Steiner St & Hayes St, San Francisco, CA 94117">Steiner St & Hayes St</option>
>                 <option value="Fisherman's Wharf, San Francisco, CA">Fisherman's Wharf</option>
>                 <option value="Golden Gate Bridge">Golden Gate Bridge</option>
>                 <option value="Presidio of San Francisco, San Francisco, CA">Presidio of San Francisco</option>
>                 <option value="Baker Beach, San Francisco, CA">Baker Beach</option>
>             </select>
>             <label for="waypoints">Locations</label>
>         </div>
>     </div>
>     <div class="col s9">
>         <div id="map" style="width: 100%"></div>
>     </div>
> </div>
> ```

Blank Section for Map

Reply:
Oh I thought you meant to put it in line, yeah I already have those

Reply:
#map {
    height: 100%;
    width: 100%;
}

#itinerary-panel{
    height: 100%;
}

Reply:
ok, try nesting the map div inside of a `<div class="card">`

Reply:
Nope, I can just push with a working version and you can see if you can get it to work on your end if you would like?

Reply:
Sure